offchain/inspect-server/src/main.rs

```diff
@@ -3,16 +3,16 @@
 
 use clap::Parser;
 
-use inspect_server::config::CLIConfig;
+use inspect_server::{config::CLIConfig, InspectServerConfig};
+use tracing::info;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    env_logger::Builder::from_env(
-        env_logger::Env::default().default_filter_or("info"),
-    )
-    .target(env_logger::fmt::Target::Stdout)
-    .init();
-    let config = CLIConfig::parse().into();
+    let config: InspectServerConfig = CLIConfig::parse().into();
+
+    logs::configure(&config.logs_config);
+
+    info!(?config, "Starting Inpect Server");
```

Comment:
```suggestion
```

This line looks like a duplicate of the following one:

https://github.com/cartesi/rollups-node/blob/e9a9c23299a79a294d2d5ad85099a8bcdae94bfe/offchain/inspect-server/src/lib.rs#L18

Reply:
@fmoura FYI